bindings/python/src/OpenSpaceToolkitPhysicsPy/Environment/Atmospheric/Earth.cpp

```diff
@@ -35,11 +35,19 @@ inline void OpenSpaceToolkitPhysicsPy_Environment_Atmospheric_Earth(pybind11::mo
             .def(
                 init<
                     const Earth::Type&,
+                    const Earth::InputDataSourceType&,
+                    const Real&,
+                    const Real&,
+                    const Real&,
                     const Shared<const Frame>&,
                     const Length&,
                     const Real&,
                     const Shared<Celestial>&>(),
                 arg("type"),
+                arg("input_data_source_type"),
+                arg("f107_constant_value") = 150.0,
+                arg("f107_average_constant_value") = 150.0,
+                arg("kp_constant_value") = 3.0,
```

Comment:
It may be cleaner to use the `Ap` scale rather than the `Kp` scale here. We use `Ap ` in the computations, and I believe `Kp` has a bunch of weird qualifiers that would be awkward to parse in the input:
https://www.ngdc.noaa.gov/stp/geomag/kp_ap.html

Reply:
Re: slack thread on this. GMAT uses Kp and it seems freeflyer allows either:
https://ai-solutions.com/_help_Files/user_created_atmospheric_drag_files.htm

Let's stick with your implementation!

Reply:
I think the first link I posted has a different definition of Kp than some other sources, where it just goes from 0-9

Reply:
Hum, interesting. I'll double check those 🙏

Reply:
Okay, following your recommendation below, closing this one